packages/telegraf/extra/telegraf.ps1

```diff
@@ -11,4 +10,25 @@ $local_ip = (Find-NetRoute -RemoteIPAddress $IPAddress | Select-Object  Ipv4Addr
 $env:DCOS_NODE_PRIVATE_IP=$local_ip;
 
 & $pkg_inst_dpath\bin\telegraf.exe --config $dcos_conf_dpath\telegraf\telegraf.conf --service install
+
+function Set-ServiceRecovery{
```

Comment:
Can we please have a comment stating what this function does, and what the parameters mean?

Why do we want 3 restart actions? Why not have an array?

Reply:
Windows Services support the ability to automatically perform some defined action in response to a failure. We  define actions (Restart the Service) that must be performed on the 1st failure, 2nd failure and configure them.

Reply:
Can you please add this description as a code comment in the script?

Why do we want the first and second failures to be different to other failures? Generally, our approach is to keep retrying (with a wait), so that services will eventually restart when thins are healthy.

Reply:
fixed at cb14fa3776e07b3c2d977a83b8fb023def069eee
I reduced code and added comments